src/screenbot.js

```diff
@@ -24,20 +30,56 @@ var Screenbot = (function Screenbot() {
              channel_token;
     };
 
-    var _request = function(endpoint, callback){
+    var _parseResponse = function(responseText){
+      try {
+        var response = JSON.parse(responseText);
+      }
+      catch (e) {
+        return _handleError({ code: ERROR_CODES.PARSE_ERROR.code });
```

Comment:
With `_handleResponse()` removed and the constants written individually, this could be:
```
return new ScreenbotError(PARSE_ERROR);
```